### PR TITLE
Release/3.2.0

### PR DESCRIPTION
**Pull request by greg-el**
New in 3.2.0:
- #1014
- #805
- #1010 
- #748
- #808 
- #1025

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: common/changes/@snowplow/browser-tracker-core/issue-808_2021-12-21-20-04.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@snowplow/browser-tracker-core",
+      "comment": "Allow alternative Access-Control-Allow-Credentials values (close #808)",
```
Comment (paulboocock) [inline common/changes/@snowplow/browser-tracker-core/issue-808_2021-12-21-20-04.json:5]: My bad 😞 
```suggestion
      "comment": "Allow alternative Access-Control-Allow-Credentials values (#808)",
```

File: common/changes/@snowplow/browser-tracker/issue-808_2021-12-21-20-04.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@snowplow/browser-tracker",
+      "comment": "Allow alternative Access-Control-Allow-Credentials values (close #808)",
```
Comment (paulboocock) [inline common/changes/@snowplow/browser-tracker/issue-808_2021-12-21-20-04.json:5]:
```suggestion
      "comment": "Allow alternative Access-Control-Allow-Credentials values (#808)",
```

File: common/changes/@snowplow/javascript-tracker/issue-808_2021-12-21-20-04.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@snowplow/javascript-tracker",
+      "comment": "Allow alternative Access-Control-Allow-Credentials values (close #808)",
```
Comment (paulboocock) [inline common/changes/@snowplow/javascript-tracker/issue-808_2021-12-21-20-04.json:5]:
```suggestion
      "comment": "Allow alternative Access-Control-Allow-Credentials values (#808)",
```

**Review by paulboocock — APPROVED** [review]
LGTM!

**Comment by shadiramadan** [comment]
@paulboocock @greg-el when is this scheduled for release to npm?

**Comment by paulboocock** [comment]
> @paulboocock @greg-el when is this scheduled for release to npm?

We're just writing the docs and then we'll release it. Will all be next week hopefully 🙏

**Pull request merged**